server/controller/login_controller.js

```diff
@@ -0,0 +1,47 @@
+/* eslint-disable linebreak-style */
+import { getSingleUser } from '../helper/userHelper';
+
+const wrongDataStatus = (res, message) => {
+  res.status(400).send({
+    status: 400,
+    message,
+  });
+};
+
+export default function login(req, res) {
+  if (req.body.email === undefined) {
```

Comment:
Use object destructuring here:
eg `const {email,password} = req.body`
then use `email` and `password` instead of `req.body.password`